### PR TITLE
Feature/Add db table for customer reviews

### PR DESCRIPTION
**Pull request by Pilyut**
Added ReviewModel, ReviewMapping and Migration

### CODE REVIEW

File: Curtains/Curtains.Domain/Data/Mappings/ReviewMapping.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+{
+    internal class ReviewMapping : IEntityTypeConfiguration<ReviewModel>
+    {
+        public void Configure(EntityTypeBuilder<ReviewModel> builder)
+        {
+            builder.ToTable(nameof(ReviewModel));
+
+            builder.Property(r => r.PersonName)
+                .HasMaxLength(10)
+                .UseCollation("SQL_Latin1_General_CP1_CI_AS");
```
Comment (EgorAntonovich) [inline Curtains/Curtains.Domain/Data/Mappings/ReviewMapping.cs]: How we can simplify using same collations instead of using it in every property?

Comment (EgorAntonovich) [reply Curtains/Curtains.Domain/Data/Mappings/ReviewMapping.cs]: May be it's make sense to move it into appsettings.json
Or we can create constants static class and use them from it.

**Review by EgorAntonovich — CHANGES_REQUESTED** [review]

File: Curtains/Curtains.Domain/Data/Mappings/ReviewMapping.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Curtains.Domain.Models
```
Comment (C0Rs1K) [inline Curtains/Curtains.Domain/Data/Mappings/ReviewMapping.cs]: I guess mappings should be in Curtains.Domain.Data namespace

**Review by C0Rs1K — CHANGES_REQUESTED** [review]

File: Curtains/Curtains.Domain/Data/Mappings/ReviewMapping.cs (hunk 2 above)
Comment (C0Rs1K) [inline Curtains/Curtains.Domain/Data/Mappings/ReviewMapping.cs]: We still need change mapping's namespaces from Curtains.Domain.Models to Curtains.Domain.Data

File: Curtains/Curtains.Infrastructure/Migrations/CurtainsContextModelSnapshot.cs [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
+using Curtains.Domain.Models;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Infrastructure;
 using Microsoft.EntityFrameworkCore.Metadata;
 using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
 
 #nullable disable
 
-namespace Curtains.Infrastructure.Migrations
+namespace Curtains.Domain.Migrations
```
Comment (C0Rs1K) [inline Curtains/Curtains.Infrastructure/Migrations/CurtainsContextModelSnapshot.cs]: this namespace is probably Curtains.Infrastructure.Migrations

**Review by C0Rs1K — CHANGES_REQUESTED** [review]
Look's good :) just rename some namespaces

File: Curtains/Curtains.Domain/Data/Mappings/ReviewMapping.cs (hunk 2 above)
Comment (C0Rs1K) [reply Curtains/Curtains.Domain/Data/Mappings/ReviewMapping.cs]: Not just ReveiwMapping but all of them)

**Review by C0Rs1K — APPROVED** [review]
Great!

**Comment by EgorAntonovich** [comment]
Please rewrite this according to new branch policy

**Pull request closed without merging**